spec/PeriodicBackgroundSync-index.bs

```diff
@@ -91,31 +93,31 @@ Privacy Considerations {#privacy}
 
-The user agent may also decide to retry each failed periodicSync event, capped to a user agent defined <dfn>maximum number of retries</dfn>.
+The user agent may also decide to [=retry=] each failed [=periodicSync event=], capped to a user agent defined <dfn>maximum number of retries</dfn>.
 
 A possible algorithm to calculate the [=time to fire=], <var>timeToFire</var> for |registration| would involve running these steps:
-1. If this is the <dfn>first attempt of firing</dfn> the periodicSync event:
+1. If this is the <dfn>first attempt of firing</dfn> the [=periodicSync event=]:
     1. Let <var>now</var> represent the current time.
     1. Let <var>siteEngagementLevel</var> represent the level of engagement the user has with the origin. Calculate <var>siteEngagementPenalty</var>, which is a factor of |siteEngagementLevel|, and is inversely proportional to it. Let <var>minPeriodicSyncInterval</var> be a global cap on time intervals between periodicSync events. Define <var>effectiveGap</var> as |siteEngagementPenalty|*|minPeriodicSyncInterval|.
     1. Let <var>minIntervalForOrigin</var> be the minimum time interval allowed by the user agent, per origin. Set <var>delayForOrigin</var> to the multiple of |effectiveGap| greater than or equal to |minIntervalForOrigin|.
-    1. Let <var>timeTillScheduledEventForOrigin</var> be the time when the next periodicSync event is scheduled for the origin, if any, null otherwise.
+    1. Let <var>timeTillScheduledEventForOrigin</var> be the time when the next [=periodicSync event=] is scheduled for the origin, if any, null otherwise.
     1. If |timeTillScheduledEventForOrigin| is not null:
         1. If |timeTillScheduledEventForOrigin| - |delayForOrigin| is greater than or equal to |minIntervalForOrigin|, abort these substeps.
         1. If |delayForOrigin| is less than or equal to |timeTillScheduledEventForOrigin|, set |delayForOrigin| to |timeTillScheduledEventForOrigin| and abort these substeps.
         1. If |delayForOrigin| is less than or equal to |timeTillScheduledEventForOrigin| + |minIntervalForOrigin|, set |delayForOrigin| to |timeTillScheduledEventForOrigin| + |minIntervalForOrigin|.
-    1. Let <var>minIntervalAcrossOrigins</var> be the minimum interval allowed between periodicSync events across origins. Let <var>timeSinceLastPeriodicSync</var> be the time since a periodicSync event was fired for any origin, if any, null otherwise.
+    1. Let <var>minIntervalAcrossOrigins</var> be the minimum interval allowed between [=periodicSync event=|periodicSync events=] across origins. Let <var>timeSinceLastPeriodicSync</var> be the time since a [=periodicSync event=] was fired for any origin, if any, null otherwise.
```

Comment:
"the minimum interval allowed between periodicSync events across origins" is undefined

Reply:
that is user agent defined, I'll add that.